Dockerfile

```diff
@@ -14,7 +14,13 @@ COPY . /app
 
 RUN python -m pip install -U pip
 
-RUN pip install -r /app/requirements.txt
+RUN python -m pip install pip-tools
+
+# update and install dev requirements
+RUN cd /app && ./update_install_dev_reqs.sh
```

Comment:
Add WORKDIR /app before and don't use cd. It should be more stable.

Reply:
OK, good to know. I'll change this.